Add title, added, removed and modified to a Commit

malun22 opened the pull request:
The Commit type does now also provide a title and a list of filepaths for added, modified and removed files.

Read more here: https://docs.gitlab.com/user/project/integrations/webhook_events/#push-events

lawvs commented:
I reviewed the documentation and noticed these fields are not returned in certain places (such as NoteEvent/BuildEvent, possibly due to outdated documentation). Maybe they should be marked as optional fields.

malun22 commented:
Sure, why not!

malun22 commented:
You are right, also on the Merge request Event the "last_commit" does not seem to also send added, modified and removed. But the title seems to be standard. But yea it's more speculation than actual knowing.

lawvs commented:
Thank you for your contribution! Version 2.1.1 has been released.
